modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -17,10 +17,15 @@ interface Item {
 const onSetupToggle = (editor: Editor, formatName: string, formatValue: string) => {
   return (api: Toolbar.ToolbarMenuButtonInstanceApi) => {
     const boundCallback = Singleton.unbindable();
+    const isNone = Strings.isEmpty(formatValue);
 
     const init = () => {
-      api.setActive(editor.formatter.match(formatName, { value: formatValue }));
-      const binding = editor.formatter.formatChanged(formatName, api.setActive);
+      // If value is empty (A None-entry in the list), check it only if the format is not set at all. Otherwise, check it if the format is set to the correct value.
```

Comment:
```suggestion
      // If value is empty (A None-entry in the list), check if the format is not set at all. Otherwise, check if the format is set to the correct value.
```